crates/typst-cli/src/compile.rs

```diff
@@ -128,6 +129,35 @@ fn export_png(document: &Document, command: &CompileCommand) -> StrResult<()> {
     Ok(())
 }
 
+fn export_svg(document: &Document, command: &CompileCommand) -> StrResult<()> {
```

Comment:
This is so parallel to export_png that I feel like there should be a shared abstraction